code/game/objects/items.dm

```diff
@@ -235,12 +237,17 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 
 		msg += "[force_str] melee damage in its current state. [throwforce_str] throwing damage in its current state."

```

Comment:
You could shorten this slightly with `msg += "[force_str] melee damage and [throwforce_str] throwing damage in its current state"`